src/components/api-request.js

```diff
@@ -49,6 +50,9 @@ export default class ApiRequest extends LitElement {
       activeSchemaTab: { type: String, attribute: 'active-schema-tab' },
       schemaExpandLevel: { type: Number, attribute: 'schema-expand-level' },
       schemaDescriptionExpanded: { type: String, attribute: 'schema-description-expanded' },
+      showAuthenticationOperationInfo: { type: String, attribute: 'show-authentication-operation-info' },
+      allowAuthenticationMultipleCall: { type: String, attribute: 'allow-authentication-multiple-call' },
+      allowAuthenticationSingleCall: { type: String, attribute: 'allow-authentication-single-call' },
```

Comment:
currently RapiDoc supports providing multiple or single API keys through the UI. Users can add or remove any number of keys using the UI. There wont be any need for any attributes.

It will also create confusion if we set `allow-authentication-multiple-call` and `allow-authentication-single-call` both to true

Moreover it will create a breaking change for #21